### ISSUE TITLE
When "enable" parameter is "false" the VS does not appear in WebUI or bigip.conf

### ISSUE THREAD

**Issue by cnicolorange** [F5Networks/f5-appsvcs-extension#523]
### Environment
 * Application Services Version: 3.31.0
 * BIG-IP Version: 15.1.4

### Summary
I create an http virtual server with an AS3 declaration (parameter "enable" is true by default).
I want to disable the VS, so I use an AS3 declaration like this:
[{ 
 "op": "add", 
 "path": "/TR1/VIP-NTRZSMU001-TR1-443001/VIP-NTRZSMU001-TR1-443001/enable",
 "value": false
}]

With this declaration the VS is disabled and when I use the GET method to retrieve the VS declarations I can see it with the parameter "enable" to "false".
My problem is that when the parameter "enable" is false I don't see anymore the virtual server in WebUI or in the file bigip.conf.
I use a tool that retrieves VS informations (address, pool members, state, etc.) with snmp get requests. As the VS doesn't appear in the configuration I can't see it in my tool.

### Steps To Reproduce
1. Create a VS
2. Submit the following declaration:
```json
[{ 
 "op": "add", 
 "path": "/TR1/VIP-NTRZSMU001-TR1-443001/VIP-NTRZSMU001-TR1-443001/enable",
 "value": false
}]
```

### Expected Behavior
When I disable a VS with tmsh command "modify ltm virtual <VSname> disabled" I can see this VS in the WebUI and in the bigip.conf (with the state disabled)
When I disable a VS with AS3 I want to have the same behavior.

### Actual Behavior
When I disable a VS with AS3 I don't see it in the WebUI and in the bigip.conf (with the state disabled)

**Comment by dstokesf5** [comment]
This is actually working as intended based on the original design of AS3. AS3 is "disabling" the VS by removing it from the configuration. I am changing this to a feature request to allow control over the virtual server enabled/disabled property.

**Comment by dstokesf5** [comment]
@cnicolorange In order to help us better understand your use case, could you elaborate on why the current behavior of removing the ltm virtual configuration does not work for you?

**Comment by mdditt2000** [comment]
Created Jira task AUTOTOOL-3294 for PM tracking

**Comment by dstokesf5** [comment]
Use case provided in duplicate issue #603:
> The use case is to disable the VIP without removing the config from BIG-IP, so that a BIG-IP admin can quickly re-enable the VIP if needed via the GUI. Re-adding the VIP using AS3 can potentially cause significant delay and unnecessary down time.

Changing the state of an AS3 configured item via the GUI is not advised as this creates a divergence between the declared state and the existing state. If the declaration is re-applied, those manual changes will be undone.

**Comment by cnicolorange** [comment]
Hello,

I answered  a few days ago but I don't see it in the issue.
I use a tool that retrieves VS informations (address, pool members, state, etc.) with snmp get requests. As the VS doesn't appear in the configuration I can't see it in my tool.

**Comment by mdditt2000** [comment]
provided a pre-release build AS3 3.39.0 build 4

**Comment by dstokesf5** [comment]
This issue was resolved in the 3.39.0 release.
